packages/playground/website/playwright/playwright.config.ts

```diff
@@ -13,7 +13,7 @@ export const playwrightConfig: PlaywrightTestConfig = {
 	/* Retry on CI only */
 	retries: process.env.CI ? 2 : 0,
 	/* Opt out of parallel tests on CI. */
-	workers: 1,
+	workers: process.env.CI ? 1 : 3,
```

Comment:
@bgrgicak, why do we opt out of parallel tests on CI, and should we continue opting out of parallel tests when running locally?

I enabled more workers to speed local testing, but we can change it back if needed.

Reply:
They were occasionally failing locally and this made them more robust. 
I just went with disabling parallel tests to get it out quicker. We can test it now with multiple workers and see how well it works.